styles/lazy-styles.css

```diff
@@ -1,5 +1,4 @@
 /* below the fold CSS goes here */
-@import url("./styles.css");
```

Comment:
So this import wasn't really needed at all here?

Reply:
> So this import wasn't really needed at all here?

Yes I am not really sure why we need to import `styles.css` here given its [already imported by head.html](https://github.com/hlxsites/sunstar/blob/759cb33b86da15d6e1349afbd13ee9ea13a623f4/head.html#L20). This comes from Sunstar Engg. project where it was added by myself 😜 but I don't know why. Likely copy/pasted from somewhere.
Removing this helps as it prevents css reload which shift the elements for a short duration while it loads, thus introducing CLS.